PlugIns/DotScene/misc/dotscene.dtd

```diff
@@ -274,3 +274,44 @@
     type ( bool | float | int | str ) "str"
     data CDATA #REQUIRED
 >
+
+<!-- https://www.w3schools.com/xml/xml_dtd_elements.asp -->
+
+<!ELEMENT animations (animation+)>
+
+<!ELEMENT animation (keyframe+)>
+<!ATTLIST animation
+	name	PCDATA			#REQUIRED
+	length	PCDATA			#REQUIRED
+	enable	(true | false)	"false"
+	loop	(true | false)	"false"
+	interpolationMode			(linear | spline)	"linear"
+	rotationInterpolationMode	(linear | spline)	"linear"
+>
+	
+<!ELEMENT keyframe (translation?, rotation?, scale?)>
+<!ATTLIST keyframe
+	time	PCDATA	#REQUIRED
+>
+
+<!ELEMENT translation EMPTY>
+<!ATTLIST translation
```

Comment:
we already have position, which is semantically the same. Therefore I would rather use that - even at the cost of incompatibility with EOE. I would rather fixup EOE.

Reply:
@paroj the problem is that this is actually a translation and not a position, see the manual:
[Ogre::TransformKeyFrame::setTranslate()](https://ogrecave.github.io/ogre/api/1.12/class_ogre_1_1_transform_key_frame.html#a8d82e03beea51388e7daf005b76b00a1)

Reply:
Nevermind, I see you already changed EOE besides it makes it easier to merge with `_ogre_node_helper` which uses position.

Reply:
I would prefer to keep the DTD simple, instead of following the API naming. It is inconsistent anyway:
position and translation are the same thing, while all values specified here are *offsets* to the previous keyframe..